changelog/17088.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+plugins: Adding version to plugin GRPC interface
```

Comment:
Formatting is slightly different for feature changelogs:

````
```release-note:feature
**Feature Name**: Description of feature - for example "Custom password policies are now supported for all database engines."
```
````

Reply:
It's more of an improvement really, so I'll just change that :)